src/nectarchain/makers/calibration/tests/test_pedestal_tool.py

```diff
@@ -166,7 +182,7 @@ def test_timesel(self):
                     pixel_mask_nevents_min=1,
                 )
 
-                tool.initialize()
+                # tool.initialize()
```

Comment:
No need to initialize the tool here ?

Reply:
I have a problem in the test with the call to `initialize`, and, if I'm write, this method is needed in case of using tools as command lines.

Reply:
OK, I see. And it is true the tests are passing as is.